rag/services/langchain_runtime.py

```
@@ -0,0 +1,42 @@
+
+def embedding_model_name() -> str:
+    return os.getenv("OPENAI_EMBEDDING_MODEL", "text-embedding-3-small")
+
+
+def chat_model_name() -> str:
+    return os.getenv("OPENAI_CHAT_MODEL", os.getenv("OPENAI_MODEL", "gpt-4o-mini"))
+
+
+@lru_cache(maxsize=4)
+def get_embeddings(model: str | None = None) -> OpenAIEmbeddings:
+    """OpenAI Embeddings（LangChain）。API キーは環境変数 OPENAI_API_KEY。"""
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)